src/steps/05-PerformanceRequirements/DOW/DOWLandingPage.vue

```diff
@@ -10,15 +10,77 @@
           v-if="(totalSections - totalSectionsComplete) !== totalSections">
           Your Performance Requirements Summary
         </h1>
+        <ATATAlert
+            id="DefiningRequirements"
+            calloutBackground="primary-lighter"
+            :showIcon="false"
+            class="container-max-width my-10 pt-6"
+            type="callout"
+        >
+          <template v-slot:content>
+            <ATATExpandableLink 
+              :hasUnderline="false"
+              :isCopyMaxWidth="false"
+              class="container-max-width mb-n2"
+              aria-id="ExpandDefiningRequirements">
+              
+              <template v-slot:header>
+                <span class="h2 pt-n1">Defining your requirements with JWCC</span> 
+              </template>
+              <template
+                  v-slot:content
+                  v-if="currentEnvironmentExists"
+              >
+                <p class="container-max-width mt-2 mb-0">
```

Comment:
There is a lot of repeated content. We try to keep things DRY as possible, even in copy. Please use the `currentEnvironmentExists` boolean in a `v-if` on the first paragraph. Also use a `v-if="!currentEnvironmentExists"` on a `<span>` in the second paragraph to toggle the sentence `JWCC provides flexibility in the way you define your requirements.` when currentEnvironment doesn't exist, and to show/hide the word `instead` in the sentence starting with "You may...". Those *look* like the only text differences - please double-check.  This will make updating/maintaining much easier.

Reply:
Oh wow, this is a MUCH better way to do this, thanks for pointing it out! I actually was trying to figure out how I could repeat less in the code, but admittedly HTML is far from my best language knowledge-wise.

Reply:
Just pushed a commit addressing this. I was able to remove all duplication by using v-if's at the <p> level and by injecting spans with v-if's where needed. Moving on to the state/store issue now.